src/routes/v2/auth.js

```diff
@@ -46,7 +51,7 @@ class AuthRouter {
     const cookieSettings = {
       expires: csrfTokenExpiry,
       httpOnly: true,
-      secure: isSecure(),
+      secure: isSecure,
```

Comment:
Oof, this seems error prone, could we define CookieSettings interface for stricter type setting? 
(I am assuming this is low effort, not sure in how many other places this object is being used in)

Reply:
HAHA just realised this JS file psps